SU-SWS/stanford_components
# EARTH-281: New variant of hero banner that includes lockup

# READY

# Summary
- Creates a variant of the hero banner with a dark "racing stripe," for department pages in Stanford Earth

# Needed By current sprint

# Steps to Test

1. modify the hero banner paragraph type:
  1. field settings: add a checkbox for racing_stripe (boolean)
  2. display settings: variant collects value from field
3. create a new complex page
4. add new hero banner, check box for the racing stripe ("include mockup")

# Affected versions or modules
- Does this PR impact any particular module, version, or pull request?  **nope.**

# Associated Issues and/or People
- JIRA ticket https://stanfordits.atlassian.net/browse/EARTH-281
- Anyone who should be notified? @sherakama 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
